### PR TITLE
added readme file

### PR DESCRIPTION
**Pull request by OSlovachevska**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+Description
```
Comment (koval-kn) [inline README.md]: replace with project name, it is obviously that readme is some kind of description

File: README.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+authentication user;
+create/get list of books;
+create/get categories of books;
+add books to a shopping cart;
+edit a shopping cart;
+checkout an order for an authenticated user;
+complete an order and get list of orders for an authenticated user;
+under role ADMIN can edit statuses of an order
+
+Technologies Used
```
Comment (koval-kn) [inline README.md]:
```suggestion
Technologies Used:
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
+Description
+
+Book store is a service which allows register customers and buy books. This API supports authentication, registration, CRUD (Create, Read, Update,Delete) operations with entity of database.
+
+Features:
```
Comment (koval-kn) [inline README.md]: make all headers in readme bold or with bigger font size

File: README.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+Liquibase 4.20.0
+
+Endpoints available for used api
+
+Available for non authenticated users:
+
+POST: /api/auth/register - register a new user,
+POST: /api/auth/login - authenticate in system.
+
+Available for users with role USER
```
Comment (koval-kn) [inline README.md]:
```suggestion
Available for users with role USER:
```

File: README.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+GET: /api/cart - get books from shopping cart,
+POST: /api/cart - add a book to shopping cart,
+PUT: /api/cart/cart-items/{cartItemId} - update book quantity in shopping cart,
+DELETE: /api/cart/cart-items/{cartItemId} - delete a book from shopping cart,
+GET: /api/orders - get list of all orders by user
+POST: /api/orders - create a new order,
+GET: /api/orders/{orderId}/items - get list of all order items by order id,
+GET: /api/orders/{orderId}/items/{itemId} - get a order item by order item id.
+
+Available for users with role ADMIN
```
Comment (koval-kn) [inline README.md]:
```suggestion
Available for users with role ADMIN:
```

File: README.md [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+DELETE: /api/categories/{id} - delete a category,
+PATCH: /api/orders/{id} - update a status of user`s 
+
+Note on use the api
+
+Please note that when using the api endpoints with methods POST, PUT, PATCH required JSON body. Security builds on technology jackson web token (JWT) with using Bearer token.
+
+Instructions:
+To run the book store API on a server, you will need to first run Docker.
+After running api database in first time will create all necessary tables in database
```
Comment (koval-kn) [inline README.md]: we cannot run API, API its just application programming interface, so replace word "api" with Application (or Docker, write here what do you think is better)

**Review by koval-kn — CHANGES_REQUESTED** [review]

**Review by sokimaaa — APPROVED** [review]

**Pull request merged**